contentgrid-testcontainers-k3s-cilium/src/main/java/com/contentgrid/testcontainers/k3s/customizer/K3sContainerCustomizers.java

```diff
@@ -0,0 +1,48 @@
+package com.contentgrid.testcontainers.k3s.customizer;
+
+import java.util.Arrays;
+import java.util.function.UnaryOperator;
+
+/**
+ * Container for {@link K3sContainerCustomizer}s
```

Comment:
This is `Container` meaning "something that contains", not "docker container", right? Maybe pick a different phrasing for it, because it's getting confusing :slightly_smiling_face: 

Same with the `@param`s in K3sContainerCustomizer.java